.travis.yml

```diff
@@ -41,7 +41,7 @@ env:
         # are in astropy-ci-extras. If your package uses either of these,
         # add the channels to CONDA_CHANNELS along with any other channels
         # you want to use.
-        # - CONDA_CHANNELS='astopy-ci-extras astropy'
+        - CONDA_CHANNELS='astropy-ci-extras astropy'
```

Comment:
Yes, exactly. Also, I think, you'll only need  to add ``CONDA_CHANNELS='astopy-ci-extras'``